code/game/objects/items/dualsaber.dm

```diff
@@ -24,7 +24,7 @@
 	light_on = FALSE
 	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
 	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
-	block_chance = 75
+	block_chance = 10 // SPLURT EDIT ANTAG_NERF, ORIGINAL BLOCK_CHANCE: 75
```

Comment:
Ok so, the vast majority of this type of changes that just edit the value of a variable are very modularizable, please modularize them rather than editing the core files